### PR TITLE
feat(global-style): remove GlobalStyle component

### PR DESCRIPTION
**Pull request by QuiiBz**
## Type

- Refactor

#### The following changes where made:

Remove `GlobalStyle` component and tests

### CODE REVIEW

**Comment by chambo-e** [comment]
Also, PR should be a `feat` to bump the package and indicate a change :)

**Comment by QuiiBz** [comment]
> Storybook seems to fail, I have an error on each page

Forgot to remove `<GlobalStyle />` from storybook, added back the css

File: .storybook/preview.js [hunk 1]
```diff
@@ -55,6 +55,99 @@ const adjustedTheme = ancestorTheme => ({
   ),
 })
 
+const globalStyles = theme => css`
```
Comment (chambo-e) [inline .storybook/preview.js:58]: Can we cut it down to the bare minimum? Eg, not sure that hiding overflow to reenable it after is optimal :D

Comment (QuiiBz) [reply .storybook/preview.js:58]: It's currently the same CSS as before, what would you want to remove apart the duplicated `overflow`?

Comment (chambo-e) [reply .storybook/preview.js:58]: I was hoping everything expect fonts 😄 

If this is a prerequisite for this ui lib to look as intended we actually should remove it ...

**Review by matthprost — APPROVED** [review]

**Review by chambo-e — APPROVED** [review]

**Pull request merged**